examples/chip-tool/commands/common/ModelCommand.cpp

```diff
@@ -205,6 +221,12 @@ void ModelCommand::ParseReadAttributeResponseCommandFailure(uint16_t clusterId,
     return;
 }
 
+void ModelCommand::HandleClusterResponse(uint16_t clusterId, uint16_t endPointId, uint16_t commandId, uint8_t * message,
+                                         uint16_t messageLen) const
+{
+    ChipLogProgress(chipTool, "Received cmd %d for cluster %d", commandId, clusterId);
```

Comment:
As we look forward to fleshing out multi-endpoint support may want to also add that to the log:

```c++
ChipLogProgress(chipTool, "Received cmd %d for cluster %d on endpoint %d", commandId, clusterId, endPointId);
```